### PR TITLE
BETA: Register nassro.is-a.dev

### PR DESCRIPTION
**Pull request by nassro199**
Added `nassro.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

File: domains/nassro.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "nassro199",
+        "email": "nasrohasini19ny@gmail.com"
+    },
+    "record": {
+        "CNAME": "nassro"
```
Comment (DEV-DIBSTER) [inline domains/nassro.json:7]: Invalid CNAME record.

**Review by DEV-DIBSTER — DISMISSED** [review]

File: domains/nassro.json (hunk 1 above)
Comment (nassro199) [reply domains/nassro.json]: > Invalid CNAME record.

i edit it sorry i was a typo

**Review by nassro199 — COMMENTED** [review]
i fixed the cname sorry for the mistake

**Review by wdhdev — CHANGES_REQUESTED** [review]
Please add content to your website.

**Review by nassro199 — COMMENTED** [review]
changed the cname

**Review by wdhdev — APPROVED** [review]

**Pull request merged**